packages/database/src/database/events.ts

```
@@ -0,0 +1,60 @@
+
+  if (query?.orgid) {
+    where = { organizationid: query.orgid }
+  }
+
+  if (query?.initid) {
+    where = { initiativeid: query.initid }
+  }
+
+  let filter = { where, skip, take, orderBy }
+  if (query?.page || query?.size) {
+    let page = parseInt(query?.page || '0')
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)